include/zfs_fletcher.h

```diff
@@ -126,6 +126,7 @@ typedef struct fletcher_4_func {
 	fletcher_4_fini_f fini_byteswap;
 	fletcher_4_compute_f compute_byteswap;
 	boolean_t (*valid)(void);
+	boolean_t uses_fpu;
```

Comment:
Yeah, I thought about that, it would squeeze the size of the struct below 64 bytes again, currently it is 72 bytes. Let me have a more thorough look tomorrow.

Reply:
I can remove `init_byteswap()/fini_byteswap()` in a follow up PR. Just wondering why there were implemented in the first place.